.size-limit.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "path": "dist/packages/ec/styles/ecl-ec.css",
-    "limit": "351 KB",
+    "limit": "377 KB",
```

Comment:
but in the end is this increased size justified? Where does it come from..?

Reply:
I guess that it comes from the utilities.
We introduced new values (and so new utilities), but we are not able to remove previous values yet.
The size should be reduced once we can clean up the styles